src/tools/recipe2plan-cli.ts

```diff
@@ -29,8 +28,11 @@ Description
   Generates Kotlin plans from recipes in a manifest. 
 
 Options
-  --outfile, -f output filename; required
+  --outfile, -f  output filename; required
   --outdir, -d  output directory; defaults to '.'
+  --format  output format, 'kotlin' or 'proto', defaults to kotlin
+  --recipeFilter  a name of the recipe to turn into a plan. If not
```

Comment:
Nit: this would more conventionally be --recipe_filter

Reply:
(I'd also suggest calling it `recipe_name`, or just `recipe`, instead. "Filter" suggests that that you might be able to use regex or wildcards or something like that)

Reply:
Changed to --recipe as recommended.